src/libcore/macros.rs

```diff
@@ -337,21 +337,16 @@ macro_rules! try {
 
 /// Write formatted data into a buffer
 ///
-/// This macro accepts a 'writer' (any value with a `write_fmt` method), a format string, and a
-/// list of arguments to format.
+/// This macro accepts a format string, a list of arguments, and a 'writer'.
+/// Arguments will be formatted according to the specified format string and the result will be
```

Comment:
without an extra `///` in between, Markdown will merge these into one paragraph; could you add some extra `///`s here and elsewhere in the file? I think you (rightfully) wanted these to be in different paragraphs 😄